src/radical/pilot/agent/launch_method/base.py

```diff
@@ -293,7 +184,7 @@ def get_rank_cmd(self):
 
     # --------------------------------------------------------------------------
     #
-    def construct_command(self, t, launch_script_hop):
+    def get_rank_exec(self, task, rank_id, rank):
```

Comment:
as a part of having one approach in method names (e.g., getting command or environment setup)

Reply:
Does that comment belong to the previous one?  If not I am missing the point I'm afraid... :-/

Reply:
*ping*  I am not sure how to interpret this comment ...

Reply:
that was a proposal of having a certain approach in method names (`.._cmd`, `.._env_setup`, `get_.._env`, etc.). But could be skipped. _Resolve this conversation_